### PR TITLE
use rich_text for event description

### LINKED ISSUE #313: Format event description
As a planner, I want to be able to format the event description. This should at least mean that links will be formatted so that they are clickable. Maybe this should mean that the description will accept Markdown.

### PR DESCRIPTION
**Pull request by jeriox**
close #313

### CODE REVIEW

File: ephios/extra/templatetags/rich_text.py [hunk 1]
```diff
@@ -15,8 +15,6 @@
     "code",
     "div",
     "em",
-    "h1",
-    "h2",
```
Comment (felixrindt) [inline ephios/extra/templatetags/rich_text.py:19]: Well actually we want these headings for pages.

Comment (felixrindt) [reply ephios/extra/templatetags/rich_text.py:19]: Can we make not having them optional?

**Review by felixrindt — CHANGES_REQUESTED** [review]

File: ephios/extra/templatetags/rich_text.py (hunk 1 above)
Comment (jeriox) [reply ephios/extra/templatetags/rich_text.py:19]: do we? as the title is already rendered as h1 i'd prefer to not have h1 in the content there.

Comment (felixrindt) [reply ephios/extra/templatetags/rich_text.py:19]: I'd opt for having them. Not having them would make `rich_text` really unintuitive.

**Review by felixrindt — APPROVED** [review]

**Pull request merged**